src/plugins/foxPage/hooks/useFarmingApr.ts

```diff
@@ -8,69 +8,77 @@ import {
   FOX_TOKEN_CONTRACT_ADDRESS,
   UNISWAP_V2_WETH_FOX_FARMING_REWARDS_ADDRESS,
   UNISWAP_V2_WETH_FOX_POOL_ADDRESS,
+  UNISWAP_V4_WETH_FOX_FARMING_REWARDS_ADDRESS,
   WETH_TOKEN_CONTRACT_ADDRESS,
 } from '../const'
 import farmingAbi from '../farmingAbi.json'
-import { getEthersProvider, rewardRatePerToken } from '../utils'
+import { getEthersProvider, makeLpApr, rewardRatePerToken } from '../utils'
 import { useCurrentBlockNumber } from './useCurrentBlockNumber'
 
 const ethersProvider = getEthersProvider()
 
 export const useFarmingApr = () => {
-  const [farmingApr, setFarmingApr] = useState<string | null>(null)
-  const [loaded, setLoaded] = useState(false)
+  const [farmingAprV2, setFarmingAprV2] = useState<string | null>(null)
+  const [farmingAprV4, setfarmingAprV4] = useState<string | null>(null)
+  const [isFarmingAprV2Loaded, setIsFarmingAprV2Loaded] = useState(false)
+  const [isFarmingAprV4Loaded, setIsFarmingAprV4Loaded] = useState(false)
   const blockNumber = useCurrentBlockNumber()
 
-  const liquidityContractAddress = UNISWAP_V2_WETH_FOX_POOL_ADDRESS
-  const uniswapLPContract = useMemo(
-    () => new Contract(liquidityContractAddress, IUniswapV2Pair.abi, ethersProvider),
-    [liquidityContractAddress],
+  const uniV2LiquidityContractAddress = UNISWAP_V2_WETH_FOX_POOL_ADDRESS
+
+  const uniV2LPContract = useMemo(
+    () => new Contract(uniV2LiquidityContractAddress, IUniswapV2Pair.abi, ethersProvider),
+    [uniV2LiquidityContractAddress],
   )
-  const farmingRewardsContract = useMemo(
+
+  const farmingRewardsV2Contract = useMemo(
     () => new Contract(UNISWAP_V2_WETH_FOX_FARMING_REWARDS_ADDRESS, farmingAbi, ethersProvider),
     [],
   )
+  const farmingRewardsContractV4 = useMemo(
```

Comment:
should we name these consistently?

Reply:
Done in 92a6de37093ea2d5dd83068398ed74ee9168c551